content/docs/notifications/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Notifications plugin"
+---
+
+Documentation section of notifications plugin.
+
+# Frontend
+The frontend provides both UI and API.  
```

Comment:
do we have frontend api? There is no mentioned in the backend section about api.

Reply:
fronend supplies api and it is in the readme. the file I added refers to that a few lines later